src/redux/actions/daoCreator.ts

```diff
@@ -22,6 +23,27 @@ export default interface DaoCreatorActions {
   ): (dispatch: Dispatch, getState: () => AppState) => Promise<void>
 }
 
+export function init(): (dispatch: Dispatch) => Promise<string> {
```

Comment:
Beautiful stuff :) I love how this is localized to the DAOCreator functionality, and isn't something we're trying to manage globally.

Less globals == more modular pieces == could use in another project